src/Microsoft.DotNet.Cli.Utils/Constants.cs

```diff
@@ -48,5 +48,7 @@ public static class Constants
         };
 
         public static readonly string MSBUILD_EXE_PATH = "MSBUILD_EXE_PATH";
+
+        public static readonly string ProjectOrSolutionArgumentName = "<PROJECT_OR_SOLUTION>";
```

Comment:
I'm not sure the underscores are needed considering you wrap it with `<>` cc: @blackdwarf